src/client/ducks/session/selectors/index.js

```diff
@@ -36,6 +36,7 @@ export const getPullRequestsWatching = createSelector(
 
 export const getPersona = state => _.get(state, ['session', 'profile', 'persona'], null)
 export const getLoggedUsername = state => _.get(state, ['entities', 'me', 'username'], null)
+export const getLoggedUser = state => _.get(state, ['entities', 'me'], null)
```

Comment:
Seems one of them is unnecessary ( `getLoggedUsername ` or `getLoggedUser `). I would drop `getLoggedUser`